src/rookify/modules/migrate_mons/main.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+
+from time import sleep
+from typing import Any, Dict
+from ..exception import ModuleException
+from ..machine import Machine
+from ..module import ModuleHandler
+
+
+class MigrateMonsHandler(ModuleHandler):
+    REQUIRES = ["analyze_ceph", "create_cluster"]
+
+    def execute(self) -> None:
+        state_data = self.machine.get_preflight_state("AnalyzeCephHandler").data
+
+        for mon in state_data["mon"]["dump"]["mons"]:
+            self._migrate_mon(mon)
+
+    def _migrate_mon(self, mon: Dict[str, Any]) -> None:
+        migrated_mons = getattr(
+            self.machine.get_execution_state("MigrateMonsHandler"), "migrated_mons", []
+        )
+        if mon["name"] in migrated_mons:
+            return
+
+        self.logger.debug("Migrating Ceph mon daemon '{0}'".format(mon["name"]))
+
+        result = self.ssh.command(
+            mon["name"], "sudo systemctl disable --now ceph-mon.target"
+        )
+
+        if result.failed:
+            raise ModuleException(
+                "Disabling original Ceph mon daemon at host {0} failed: {1}".format(
+                    mon["name"], result.stderr
+                )
+            )
+
+        self.logger.debug(
+            "Waiting for disabled original Ceph mon daemon '{0}' to disconnect".format(
+                mon["name"]
+            )
+        )
+
+        while True:
```

Comment:
We should check, if librados has a timeout for a mon connection, and automatically switches over to another mon, as we are probably shutting down the mon we are connected to.